requirements.txt

```diff
@@ -0,0 +1,18 @@
+Pillow==3.1.0
```

Comment:
I know it seems huge, but `scikit-img` drains `numpy`, which in turn drains the whole world... I went through the process of building a full virtualenv from scratch precisely to build this "short"-list of dependencies, and that's the result.